### ISSUE TITLE
2spooky4me: Double spooky potions

### ISSUE THREAD

**Issue by eiron** [HabitRPG/habitica#6048]
Probably this has only happened to myself, but I bought a full set of 18 spooky potions, went away from the market and over to the tavern to book myself out of the inn, then reloaded the data on the data display tool. In the midst of this action, I noticed my gems plummet (and 180 gold reappear, which it turns out was from buying gems for gold failing previously). Buying back those gems still put me nowhere near my original count and - here we get to the crux of the matter - I have 36 spooky potions (twice what I purchased). Any chance I can have the gems reinstated rather than being stuck with all this superfluous spookiness?

User ID: 4182cdea-64db-4db4-9713-dc587d86c662

tl;dr - Bought spooky potions. Ended up with double spooky potions. Oh no!

**Comment by crookedneighbor** [comment]
I've removed the potions and adjusted your gems.
